.github/workflows/build_steps.yml

```diff
@@ -120,22 +120,22 @@ jobs:
           buildPreset: ${{env.ARCTIC_CMAKE_PRESET}}
 
       - name: Compile C++ tests
-        if: inputs.job_type == 'cpp-tests'
+        if: inputs.job_type != 'follower'
         run: cd cpp; cmake --build --preset $ARCTIC_CMAKE_PRESET --target install
 
-      - name: Clean up build dir # To save disk space as we're close to the limit on Linux
-        if: inputs.job_type == 'cpp-tests' && matrix.os == 'linux'
-        run: |
-          cd $ARCTICDB_BUILD_DIR/*-build
-          du -m --max-depth=2 | sort -n | tail -n 50
-          nohup rm -rf * &  # Clean up while the test is running to save build time
+      # - name: Clean up build dir # To save disk space as we're close to the limit on Linux
+      #   if: inputs.job_type != 'follower' && matrix.os == 'linux'
+      #   run: |
+      #     cd $ARCTICDB_BUILD_DIR/*-build
+      #     du -m --max-depth=2 | sort -n | tail -n 50
+      #     nohup rm -rf * &  # Clean up while the test is running to save build time
 
       - name: C++ Rapidcheck
-        if: inputs.job_type == 'cpp-tests'
+        if: inputs.job_type != 'follower'
```

Comment:
Avoid bundling the tests with the compilation:

- If any of the tests fail spuriously and the user wants just to rerun the test, they'd end up having to rerun everything.
- If one test fails, it will fail the rest of the build, so the user will not have a chance to learn what other tests will fail, esp. for other Python versions, which actually makes the feedback cycle longer. (Note our tests tend to seg fault the entire test runner.)
- Also, some tests are meant to take some time (hypothesis, stress), so running them here will block the followers.